app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -461,6 +495,29 @@ def schedule_set_record_vars(schedule)
       uri_settings[:name] = params[:depot_name]
       uri_settings[:save] = true
       schedule.verify_file_depot(uri_settings)
+    elsif params[:action_typ] == "automation_request"
+      attrs = []
+      AE_MAX_RESOLUTION_FIELDS.times do |i|
+        attrs[i] = []
+        attrs[i][0] = params[:attrs][i.to_s][0] if params[:attrs] && params[:attrs][i.to_s]
```

Comment:
@h-kataria - Is there a way to pull out some of the duplication between line 502 and 503?